docs/index.rst

```diff
@@ -229,6 +229,8 @@ Ready to start reviewing pull requests? We have :doc:`a guide </reviewing_PRs>`
 for that too! Finally, the :doc:`API reference </api/index>` covers individual
 functions, classes, methods and modules.
 
+**Robottelo** is compatible with Python 2.7.
```

Comment:
I use robottelo only with Python 3.4 (currently installed on my machine) it works fine with both Python 2 and 3. Also it is checked on travis for both versions.